solutions/usaco-1037.mdx

```diff
@@ -14,6 +14,99 @@ By simulating handshakes for all values of $K$, we can find the minimum, maximum
 
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+using vi = vector<int>;
+#define all(x) begin(x), end(x)
+#define sz(x) (int)(x).size()
```

Comment:
can you detemplify the code plz? (optional)

Reply:
done!